store/src/java/com/zimbra/cs/service/admin/AdminDocumentHandler.java

```diff
@@ -588,10 +588,8 @@ protected Element proxyIfNecessary(Element request, Map<String, Object> context)
             if (acctId != null) {
                 Account acct = getAccount(prov, AccountBy.id, acctId, zsc.getAuthToken());
                 if (acct != null && !Provisioning.onLocalServer(acct, reasons)) {
-                    if (zsc.getHopCount() > 2 || (ZimbraLog.soap.isDebugEnabled())) {
-                        ZimbraLog.soap.info("Proxying request:ProxiedAccountPath=%s reasons:%s",
-                                Joiner.on("/").join(xpath), reasons.getReason());
-                    }
+                    ZimbraLog.soap.info("Proxying request:ProxiedAccountPath=%s reasons:%s",
```

Comment:
nit: I prefer adding a space after the colon (```:```) for both ```Proxying request:[space]``` and ```reason:[space]``` (note, I'm also suggesting we also just use *reason* (singular) instead of *reasons* (plural) regardless of whatever follows that) on this and each of the other similar log events in this change.

Reply:
adding a space and making "reasons" singular -> Done